tox.ini

```
@@ -25,3 +25,7 @@ ignore=
 [coverage:run]
 branch=True
 source=api,tests
+
+[isort]
+profile=black
+multi_line_output=3
```

```suggestion
profile=black
```
This isn't needed as it's part of the default black profile